### PR TITLE
Fixing build issues with limits

### PR DESCRIPTION
**Pull request by codyseibert**
we are hitting a lot of AWS cloudformation limits; so we are splitting our api into 2 serverless files and deploying them separate.

notable changes:
- the dynamo and s3 serverless plugins are no longer used; instead, we use the underlying dynamo jar and s3rver module and host them via .sh / node scripts
- the api service is hosted on port 3001, the cases service is on port 3002, there is a proxy on port 3000 which maps to the local serverless functions so the UI can still hit a single endpoint
- removing v1 from all endpoints

### CODE REVIEW

**Review by rachaelparris — APPROVED** [review]

File: web-client/src/applicationContext.js [hunk 1]
```diff
@@ -236,7 +236,7 @@ tryCatchDecorator(allUseCases);
 
 const applicationContext = {
   getBaseUrl: () => {
-    return process.env.API_URL || 'http://localhost:3000/v1';
+    return process.env.API_URL || 'http://localhost:3000';
```
Comment (kkoskelin) [inline web-client/src/applicationContext.js:239]: why aren't we adding `/api` here instead of everywhere else?

File: docker-zap.sh [hunk 2]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 CONTAINER_NAME="dockerzap"
 set +e
-docker run --name "${CONTAINER_NAME}" -v $(pwd):/zap/wrk/:rw -t owasp/zap2docker-weekly zap-api-scan.py -t https://efcms-dev.ustc-case-mgmt.flexion.us/v1/swagger.json -f openapi -d -g gen.conf -r zap-report.html -z "-configfile /zap/wrk/options.prop"
+docker run --name "${CONTAINER_NAME}" -v $(pwd):/zap/wrk/:rw -t owasp/zap2docker-weekly zap-api-scan.py -t https://efcms-dev.ustc-case-mgmt.flexion.us/swagger.json -f openapi -d -g gen.conf -r zap-report.html -z "-configfile /zap/wrk/options.prop"
```
Comment (kkoskelin) [inline docker-zap.sh]: Does this swagger url also need `/api` in place of `/v1`?

File: web-api/run-serverless-cases.sh [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+USER_POOL_ID=$(aws cognito-idp list-user-pools --query "UserPools[?Name == 'efcms-${slsStage}'].Id | [0]" --max-results 30 --region "us-east-1")
+USER_POOL_ID="${USER_POOL_ID%\"}"
+USER_POOL_ID="${USER_POOL_ID#\"}"
+
+ACCOUNT_ID=$(aws sts get-caller-identity --query "Account")
+ACCOUNT_ID="${ACCOUNT_ID%\"}"
+ACCOUNT_ID="${ACCOUNT_ID#\"}"
+export NODE_PRESERVE_SYMLINKS=1
+cd src && find . -type f -exec chmod -R ugo+r {} ";"
```
Comment (kkoskelin) [inline web-api/run-serverless-cases.sh]: could simplify to `find ./src ...` and then omit the `cd ..` which follows.

File: web-api/run-serverless.sh [hunk 4]
```diff
@@ -14,9 +14,14 @@ export NODE_PRESERVE_SYMLINKS=1
 cd src && find . -type f -exec chmod -R ugo+r {} ";"
 cd ..
 npm run build
+cpp src/handlers.js /tmp
```
Comment (kkoskelin) [inline web-api/run-serverless.sh]: `cpp`?  or `cp`?

File: web-api/run-serverless.sh [hunk 5]
```diff
@@ -14,9 +14,14 @@ export NODE_PRESERVE_SYMLINKS=1
...
 cp dist/* src
+
 SLS_DEPLOYMENT_BUCKET="${EFCMS_DOMAIN}.efcms.${slsStage}.${region}.deploys"
 SLS_DEPLOYMENT_BUCKET="${SLS_DEPLOYMENT_BUCKET}" ./node_modules/.bin/sls create_domain --stage "${slsStage}" --region "${region}" --domain "${EFCMS_DOMAIN}" --userPoolId "${USER_POOL_ID}" --efcmsTableName="efcms-${slsStage}" --accountId "${ACCOUNT_ID}" --verbose
 
 ENVIRONMENT="${slsStage}" SLS_DEPLOYMENT_BUCKET="${SLS_DEPLOYMENT_BUCKET}" ./node_modules/.bin/sls deploy --stage "${slsStage}" --region "${region}" --domain "${EFCMS_DOMAIN}"  --userPoolId "${USER_POOL_ID}" --verbose --efcmsTableName="efcms-${slsStage}" --accountId "${ACCOUNT_ID}"
 ./configure-custom-api-access-logging.sh "${slsStage}" ./config-custom-access-logs.json "${region}"
+
+cp src/handlers.js src
```
Comment (kkoskelin) [inline web-api/run-serverless.sh]: The source and destination are the same?

File: web-api/seed-dynamo.js [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+const seedEntries = require('./storage/fixtures/seed');
+
+const client = new AWS.DynamoDB.DocumentClient({
+  endpoint: 'http://localhost:8000',
+  region: 'us-east-1',
+})
+
+const main = async () => {
+  for (const item of seedEntries) {
+    await client.put({
```
Comment (kkoskelin) [inline web-api/seed-dynamo.js]: unless there is some particular order we need to maintain (according to filenames?) we could return `Promise.all(entries.map(unawaitedClientPutFuncRef))`
which would speed things up.

**Review by kkoskelin — CHANGES_REQUESTED** [review]
Happy to be corrected if there are things I'm not getting (which there surely are!)

File: web-client/src/applicationContext.js (hunk 1 above)
Comment (codyseibert) [reply web-client/src/applicationContext.js:239]: currently, /api will be for all routes other than /cases routes... so it basically becomes

http://localhost:3000/api/some/route
http://localhost:3000/cases/{caseId}

The idea is that as we get more routes, we might need to split off another serverless file for maybe 

http://localhost:3000/work-items or something, but the overhead of having each resource it's own serverless file seems too much to do that now

**Review by kkoskelin — APPROVED** [review]

**Pull request merged**